File: public/programs/cpp/p10.cpp
// Print day when n given using switch case

#include<iostream.h>
using namespace std;

int main()
{
	int n;
	cout<<"Enter the number"; 
	cin>>n;
	switch(n)                                // switch case
	{
		default:cout<<"Invalid";break;       // default statement given first in switch case saves cost of execution in worst case scenario
		case 1:cout<<"Sunday";break;
		case 2:cout<<"Monday";break;
		case 3:cout<<"Tuesday";break;
		case 4:cout<<"Wednesday";break;
		case 5:cout<<"Thursday";break;
		case 6:cout<<"Friday";break;
		case 7:cout<<"Saturday";break;
		
	}
}

	

File: public/programs/java/p10.java
import java.util.*;
class p10
{
	public static void main(String args[])
	{
		int a;
		Scanner scan=new Scanner(System.in);
		System.out.print("Enter a number: ");
		a=scan.nextInt();
		switch(a)
		{
           case 1:
              System.out.print("Sunday");
              break;
           case 2:
              System.out.print("Monday");
              break;
            case 3:
              System.out.print("Tuesday");
              break;
            case 4:
              System.out.print("Wednesday");
              break;
             case 5:
              System.out.print("Thursday");
              break;
             case 6:
              System.out.print("Friday");
              break;
             case 7:
              System.out.print("Saturday");
              break;
             default:
              System.out.print("Invalid Input");               
 
		}
	}
}		